2-api/clients.md

```diff
@@ -123,8 +123,8 @@ See the docs [on the project's GitHub page](https://github.com/sloria/perspectiv
 Read more about the Perspective API and how to use it.
 
 * [Key concepts](key-concepts.md)
-* [Models and attributes](models.md)
-   * [Model cards](model-cards/README.md)
+* [Attributes](models.md)
+   * [Attribute cards](model-cards/README.md)
```

Comment:
Let's keep this as "model cards", since this is a name we've used in external comms and internal conversations

Reply:
This is still not resolved--I see attribute cards here, I only see the change in the other location.